### PR TITLE
Self describing 180 0.6.0

### PR DESCRIPTION
**Pull request by edwinvandenbelt**

### CODE REVIEW

**Comment by ARBaart** [comment]
This PR is into master, I think it should be into a develop branch.

File: TOMP-API.yaml [hunk 1]
```diff
@@ -658,6 +657,29 @@ paths:
...
+      summary: describes the running implementations
+      description: all versions that are implemented on this url, are described in the result of this endpoint. In contains all versions and per version the endpoints, their status
+        and the supported scenarios.
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
```
Comment (ARBaart) [inline TOMP-API.yaml:673]: Why is this an array? endpointImplementation describes one operator right?

File: TOMP-API.yaml [hunk 2]
```diff
@@ -658,6 +657,29 @@ paths:
         '404':
           $ref: '#/components/responses/404NotFound'
 
+  /operator/meta:
+    parameters:
+      - $ref: '#/components/parameters/acceptLanguage'
```
Comment (ARBaart) [inline TOMP-API.yaml:662]: In the most recent version (0.5.3) we switched to double quotes

File: TOMP-API.yaml (hunk 1 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:673]: nope. An operator can serve multiple versions. maybe 'endpointVersion' should be a better word.

**Comment by edwinvandenbelt** [comment]
I'll change it and will point it to major branche

**Pull request closed without merging**